test/onnx/test_onnxscript_runtime.py

```diff
@@ -126,5 +129,37 @@ def custom_layer_norm(
         self.run_test(model, (x, y, z))
 
 
+class TestATenlibtRuntime(onnx_test_common._TestONNXRuntime):
+
+    # This class shows how we can utilize existing ATenLib on torch.onnx.export
+    # torch.onnx.export relies on multiple "ONNX C++ passes" applied to its node-level
+    # graph, but the passes are not compatible with onnxscript functions. Therefore, to
+    # leverage the existing passes with torch.onnx.export, we embedds ATenLib into
+    # node-level graph as a custom operator.
+
+    # NOTE: opset18 is the only supported by ATenLib now (latest ORT supporting version).
+    opset_version = 18
+
+    def test_atenlib_selu(self):
+
+        x = torch.randn(1, 2, 3, 4, requires_grad=True)
+        model = torch.nn.SELU()
+
+        # function_dispatcher is experimental and subject to change,
+        # but how an OnnxFunction is written can be referenced
+        Selu = function_dispatcher._ATENLIB_ONNX_FUNCTIONS.get("aten::selu", None)
+        self.assertIsNotNone(Selu)
+
+        def custom_selu(g: jit_utils.GraphContext, X):
+            return g.onnxscript_op(Selu, X).setType(X.type())
+
+        torch.onnx.register_custom_op_symbolic(
```

Comment:
I think this part can be moved inside exporter, to register for ops when no old style symbolic function is found.

I wonder what's a good way to test this.

Reply:
I remember when I implemented g.onnxscript_op() I did think about how we could avoid symbolic_fn, and only have onnx-script function. However, for ops (we say trace_only=True now), need parse_args:
https://github.com/pytorch/pytorch/blob/3f840cc627b8e0002cdf6593e9d1d83a46255f31/test/onnx/test_onnxscript_runtime.py#L110-L118

I guess we can have another way to apply the trace_only=False ops in ATenLib, for example, user only needs to register it with the function as long as the function doesn't need any preprocessing on args. But in that case, there will be two ways to engage with onnx-script, and it's more complicated. What do you think?